middleware/hosts/testdata/case-hosts

```diff
@@ -0,0 +1,2 @@
+127.0.0.1	PreserveMe	PreserveMe.local
```

Comment:
Can we put these files into the source and make `Hostsfile` read from the strings? `strings.NewReader` or similar can be used.

We currently don't have any test data artifacts in the coredns code base which is something I like. Look in the file middleware to see how it is done.